pkg/config/apm.go

```diff
@@ -16,6 +16,9 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
+// Traces type used for Vector override
```

Comment:
```suggestion
// Traces specifies the data type used for Vector override.
```
What is Vector? If the reader doesn't know, can you provide some insight here?